po/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ GETTEXT_CREATE_TRANSLATIONS(exiv2.pot ALL
     fr.po
     gl.po
     ms.po
+    nl.po
```

Comment:
This change should be part of #927

Reply:
Yes.  We should only merge README-SAMPLES.md to '0.27-maintenance' and 'master' from this PR.

I've asked Paul (the contributor of #927) to change po/CMakeLists.txt and to deal with the missing translations in po/nl.po.

Reply:
I've dropped this line from the PR.